PRISM_maproom/widgets.py

```diff
@@ -0,0 +1,75 @@
+import dash_core_components as dcc
```

Comment:
I don't want a separate copy of this file in each application. For now they can share it the same way they share pingrid.py, namely it lives in the parent directory and each application has a symlink to it.

Reply:
@drewmresnick please make a note of this to address in the next PR.